sympy/physics/control/lti.py

```diff
@@ -1941,3 +2494,18 @@ def elem_zeros(self):
     def _flat(self):
         """Returns flattened list of args in TransferFunctionMatrix"""
         return [elem for tup in self.args[0] for elem in tup]
+
+    def _eval_evalf(self, prec):
+        """Calls evalf() on each transfer function in the transfer function matrix"""
+        mat = self._expr_mat.applyfunc(lambda a: a.evalf(prec))
+        return _to_TFM(mat, self.var)
```

Comment:
The prec passed to `_eval_evalf` is binary but you are passing it as the decimal prec to `evalf`.